app/webutil.js

```diff
@@ -119,7 +119,7 @@ export function initSettings (callback /*, ...callbackArgs */) {
     "use strict";
     const callbackArgs = Array.prototype.slice.call(arguments, 1);
     if (window.chrome && window.chrome.storage) {
-        window.chrome.storage.sync.get(function (cfg) {
+        window.chrome.storage.sync.get((cfg) => {
             settings = cfg;
             if (callback) {
                 callback.apply(this, callbackArgs);
```

Comment:
`this` will change value here. Have you checked the callers and made sure they are unaffected?

Reply:
Yes. The context is never used in the callbacks use through the application

This change also makes the scope of the callback consistent regardless of whether chrome storage is used or not. Before it was different.